src/routes/UserRecordContainer.js

```diff
@@ -158,8 +158,7 @@ class UserRecordContainer extends React.Component {
     },
     settings: {
       type: 'okapi',
-      records: 'configs',
-      path: 'configurations/entries?query=(module==USERS and configName==profile_pictures)',
+      path: 'users/configurations/entry',
```

Comment:
@Terala-Priyanka, it appears this endpoint is added in `users` interface `16.1` but this query isn't fenced off by something that checks for that interface. This means the Q release either needed to drop support for `16` and bump its major version to `11`, or this work should have been properly fenced off by interface version checks. Do you know which is correct?